edgraph/server.go

```diff
@@ -827,7 +827,10 @@ func buildUpsertQuery(qc *queryContext) string {
 	}
 
 	qc.condVars = make([]string, len(qc.req.Mutations))
-	upsertQuery := strings.TrimSuffix(qc.req.Query, "}")
+
+	var b strings.Builder
+	b.WriteString(strings.TrimSuffix(qc.req.Query, "}"))
```

Comment:
Thanks for addressing the comments. One more small comment.
`WriteString` returns an error. Please add x.Check2(..) that validates the error as well. We can merge this after that.

Reply:
@Phill240 please address this too. ^

Reply:
Done. Now all the WriteString methods are wrapped in Check2.